styles.css

```diff
@@ -0,0 +1,29 @@
+.container {
+  display: flex;
+  background-color: purple;
+  flex-direction: column;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+
+}
+.paths {
+  text-align: center;
+}
+.row {
+  display: flex;
+  flex-direction: row;
+}
+
+.one {
+
+  padding: 20px;
+}
+.two {
+padding: 20px
+}
+h2 {
+  flex: 1;
+  padding:20px;
+}
+h1 {
+  padding: 20px;
+}
```

Comment:
Looks like this isn't being used. Always clean up your code before finalizing your PR.